scripts/install.sh

```diff
@@ -78,12 +79,18 @@ verifySupported() {
 
 # getLatestVersion gets the latest release version.
 getLatestVersion() {
-    # Get tag from release URL
-    local latest_release_url='https://github.com/konveyor/move2kube/releases'
+    # Get tag from releaseinfo.json
+    local json_data=''
+    local release_info_url='https://raw.githubusercontent.com/konveyor/move2kube/gh-pages/_data/releaseinfo.json'
```

Comment:
Is there no way to access the url using konveyor.io url as github pages?

Reply:
why is that important?